.travis.yml

```diff
@@ -10,21 +10,21 @@ os:
 - linux
 
 go:
-  - "1.12.x"
+  - "1.13.4"
```

Comment:
Changed this to be an explicit version as well; thought that if we'd have to update the AppVeyor config anyway, so might as well pin both travis and appveyor to use an explicit version